src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -509,6 +509,16 @@ private function addSessionSection(ArrayNodeDefinition $rootNode)
                             ->defaultValue(0)
                             ->info('seconds to wait between 2 session metadata updates')
                         ->end()
+                        ->integerNode('sid_length')
+                            ->defaultValue(32)
+                            ->min(22)
+                            ->max(256)
+                        ->end()
+                        ->integerNode('sid_bits_per_character')
+                            ->defaultValue(5)
+                            ->min(4)
+                            ->max(6)
```

Comment:
please add `->info()` node which explains what it does (and maybe why it could be useful?)

Reply:
see my comment on the previous one